CMakeLists.txt

```diff
@@ -640,6 +640,7 @@ foreach(d ${gt_src_list})
           COMMAND ${CMAKE_COMMAND} -E copy ${YDB_SOURCE_DIR}/${d}/${m} "${m_out}"
           )
         list(APPEND files_to_place "${m_out}")
+	list(APPEND m_routines "${m_out}")
```

Comment:
The indention appears to be off here

Reply:
Thanks; fixed. It looks like Vim decided to use tabs on my behalf; most of the file is spaces. I reverted to spaces and will fix up the remaining tabs.